.github/workflows/velox_weekly.yml

```diff
@@ -99,3 +100,86 @@ jobs:
           cd $GITHUB_WORKSPACE/ && ./dev/package.sh
+
+  build-on-openeuler:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ "openeuler/openeuler:24.03" ]
+        spark: [ "spark-3.5" ]
+        java: [ "java-8" ]
+    if: ${{ startsWith(github.repository, 'apache/') }}
+    runs-on: ubuntu-24.04-arm
+    container: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v4
+      - name: build
+        run: |
+          dnf update -y && dnf install -y java-1.8.0-openjdk-devel sudo patch wget git perl
+          $SETUP install_maven
+          export JAVA_HOME=/usr/lib/jvm/java-1.8.0-openjdk
+          export MAVEN_HOME=/usr/lib/maven
+          export PATH=${MAVEN_HOME}/bin:${JAVA_HOME}/bin:${PATH}
+          cd $GITHUB_WORKSPACE/ && ./dev/package.sh
+          cp package/target/thirdparty-lib/gluten-thirdparty-lib-*.jar /root/
+      - name: Set environment variables
+        run: |
+          echo "JAVA_HOME=/usr/lib/jvm/java-1.8.0-openjdk" >> $GITHUB_ENV
+      - name: Build gluten-it
+        run: |
+          echo "JAVA_HOME: $JAVA_HOME"
+          cd $GITHUB_WORKSPACE/
+          $MVN_CMD clean install -P${{ matrix.spark }} -P${{ matrix.java }} -Pbackends-velox -DskipTests
+          cd $GITHUB_WORKSPACE/tools/gluten-it
+          $MVN_CMD clean install -P${{ matrix.spark }} -P${{ matrix.java }}
+      - name: Run TPC-H
+        run: |
+          echo "JAVA_HOME: $JAVA_HOME"
+          cd $GITHUB_WORKSPACE/tools/gluten-it
+          cp /root/gluten-thirdparty-lib-*.jar package/target/lib/
+          GLUTEN_IT_JVM_ARGS=-Xmx5G sbin/gluten-it.sh queries-compare \
+            --local --preset=velox --benchmark-type=h --error-on-memleak --off-heap-size=10g -s=1.0 --threads=16 --iterations=1 \
+            --extra-conf=spark.gluten.loadLibFromJar=true \
+            --extra-conf=spark.gluten.loadLibOS=openEuler --extra-conf=spark.gluten.loadLibOSVersion=24.03
+
+  build-on-openeuler-vcpkg:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ "openeuler/openeuler:24.03" ]
+        spark: [ "spark-3.5" ]
+        java: [ "java-8" ]
+    if: ${{ startsWith(github.repository, 'apache/') }}
+    runs-on: ubuntu-24.04-arm
+    container: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v4
+      - name: build
+        run: |
+          dnf update -y && dnf install -y java-1.8.0-openjdk-devel sudo patch wget git perl
+          $SETUP install_maven
+          export JAVA_HOME=/usr/lib/jvm/java-1.8.0-openjdk
+          export MAVEN_HOME=/usr/lib/maven
+          export PATH=${MAVEN_HOME}/bin:${JAVA_HOME}/bin:${PATH}
+          export VCPKG_PATH=/var/cache/vcpkg
+          export VCPKG_BINARY_SOURCES="clear;files,${VCPKG_PATH},readwrite"
+          cd $GITHUB_WORKSPACE/ 
+          ./dev/vcpkg/setup-build-depends.sh
+          mkdir -p ${VCPKG_PATH}
+          sed -i 's/--build_arrow=OFF/--build_arrow=ON/' dev/package-vcpkg.sh && ./dev/package-vcpkg.sh
```

Comment:
Can we just use `./dev/package-vcpkg.sh --build_arrow=ON`? It may override build arrow setting.